CONTRIBUTING.md

```diff
@@ -0,0 +1,198 @@
+# Contributing to Extension Activity Monitor
+
+Firstly, thank you for your interest in contributing to Extension Activity Monitor. Your contribution will make this project more awesome. 🚀
+
+## Table of Contents
+
+- [Project Architecture](#project-architecture)
+  - [Some Important Components](#some-important-components)
+    - [Background Page](#background-page)
+    - [Popup Page](#popup-page)
+    - [Activity Log Page - Tab](#activity-log-page---tab-extension-page)
+    - [Activity Log Page - Devtools](#activity-log-page---devtools-extension-page)
+  - [Core Features](#core-features)
+    - [Collecting Logs](#collecting-logs)
+      - [Live Logging](#live-logging)
+      - [Loading / saving logs](#loading--saving-logs)
+    - [Rendering Logs](#rendering-logs)
+    - [Filtering Log Entries](#filtering-log-entries)
+- [Picking an issue](#picking-an-issue)
+- [Installation](#installation)
+- [Writing and Running Tests](#writing--running-tests)
+  - [Writing a test](#writing-a-test)
+- [Checking for Linter](#checking-for-linter)
+- [Checking for Prettier](#checking-for-prettier)
+- [Submitting a Bug or Issue or Feature Request](#submitting-a-bug-or-issue-or-feature-request)
+- [Creating a Pull Request](#creating-a-pull-request)
+  - [Writing Commit Message](#writing-commit-message)
+
+## Project Architecture
+
+Extension Activity Monitor is composed by a backend part (running in the background page) and a frontend part (composed by a browserAction Popup and an Activity Log extension page which is loaded as a tab or as a devtools panel).
+
+### Some Important Components
```

Comment:
Replace "Some Important Components" with "The Extension Pages", and remove `(Extension Page)` from some subsections below.